content/articles/getting-started-with-nltk-wordnet-in-python/index.md

```diff
@@ -0,0 +1,240 @@
+
+### Prerequisites
+To have a better understanding of this article, the reader should:
+- Have basic knowledge of the python language.
+- Have [python](https://www.python.org/downloads/) installed.
+- Have [nltk](https://www.nltk.org/install.html) and its [corpus](https://www.nltk.org/data.html) installed.
+ 
+### Table of contents
+- [A start with Synonyms and Synsets](#a-start-with-synonyms-and-synsets)
+- [Understanding NLTK Hypernyms and Hyponyms](#understanding-nltk-hypernyms-and-hyponyms)
+- [A look into Meronyms and Holonyms](#a-look-into-meronyms-and-holonyms)
+- [Understanding NLTK Entailments](#understanding-nltk-entailments)
+- [Conclusion](#conclusion)
+
+### A start with Synonyms and Synsets
+WordNet categorizes English words into groups of synonyms, which are referred to as synsets (short for set of synonyms). Every Synset contains a name, a part-of-speech ( include nouns, verbs, adverbs, and adjectives) and a number. Synsets are used to store synonyms, where each word in the synset shares the same meaning . Essentially, each synset is a collection of synonyms. Some of the words have just one Synset, while others have multiple Synsets.  Every synset has a definition associated with it. Synset make it easier for users to look up words in the WordNet database.
+
+#### 1. Getting Synsets.
+Synsets are words related to the word that you entered as an argument to wordnet.synsets("word")  can be obtained from a database using this method.
+
+```python
+import nltk
+from nltk.corpus import wordnet as wn 
+wn.synsets('book')
+```
+
+OUTPUT
+```bash
+[Synset('book.n.01'),Synset('book.n.02'),Synset('record.n.05'),Synset('script.n.01'),Synset('ledger.n.01'),Synset('book.n.06'),Synset('book.n.07'),Synset('koran.n.01'),Synset('bible.n.01'),Synset('book.n.10'),Synset('book.n.11'),Synset('book.v.01'),Synset('reserve.v.04'),Synset('book.v.03'),Synset('book.v.04')]
+```
+The function also allows you to restrict the word's part of speech by providing an optional position argument.
+
+Example for verbs
+```python
+import nltk
+from nltk.corpus import wordnet as wn
+wn.synsets('book', pos=wn.VERB)
+```
+OUTPUT
+```bash
+
+[Synset('book.v.01'),
+ Synset('reserve.v.04'),
+ Synset('book.v.03'),
+ Synset('book.v.04')]
+ 
+```
+Example for nouns
+```python
+import nltk
+from nltk.corpus import wordnet as wn
+wn.synsets('book', pos=wn.NOUN)
+```
+OUTPUT
+```bash
+[Synset('book.n.01'),Synset('book.n.02'),Synset('record.n.05'),Synset('script.n.01'),Synset('ledger.n.01'),Synset('book.n.06'),Synset('book.n.07'),Synset('koran.n.01'),Synset('bible.n.01'),Synset('book.n.10'),Synset('book.n.11')]
+```
+#### 2.  Getting the definition of a Synset.
+To do this, you can utilize the definition() function, which can be used to further analyze the Synset for a definition that is common to all of its Lemmas. This method returns a string that conforms to the most basic specification. There are two ways of achieving this:

```

Comment:
Let the statement be complete from the agreeable sentence structure. "To get the definition of  a synset...."